tls/s2n_alerts.c

```diff
@@ -60,6 +60,8 @@ static S2N_RESULT s2n_translate_protocol_error_to_alert(int error_code, uint8_t
          */
         S2N_ALERT_CASE(S2N_ERR_NO_RENEGOTIATION, S2N_TLS_ALERT_HANDSHAKE_FAILURE);
 
+        S2N_ALERT_CASE(S2N_ERR_KTLS_KEYUPDATE, S2N_TLS_ALERT_UNEXPECTED_MESSAGE);
```

Comment:
All protocol errors need an alert translation. And we are saying that we don't expect KeyUpdate messages with ktls, so they are "unexpected".

What other error type would you suggest? The sending error made sense to me as a usage error, because it's triggered by application actions (sending too much application data). But the receiving version is triggered by peer actions (sending the KeyUpdate), which makes me think protocol error.

Reply:
Eh, sounds like we don't have a better error type than protocol error. I agree, it's not a usage error.